### PR TITLE
added some notes, started working on creating a selectedTasks function

### PR DESCRIPTION
**Pull request by leighstephenson**
-in saga, but want to discuss with group before doing more

### CODE REVIEW

File: src/redux/sagas/task.saga.js
Comment (sjg3609) [inline src/redux/sagas/task.saga.js:1]: Can we review line 33? Is the syntax for the server path correct?

**Review by sjg3609 — CHANGES_REQUESTED** [review]
Looks good! Sort of. Check comments. pls n thx

File: src/redux/sagas/task.saga.js
Comment (leighstephenson) [reply src/redux/sagas/task.saga.js:1]: I believe so!! Correct me if I'm wrong!

**Review by leighstephenson — COMMENTED** [review]
See comment

**Review by sjg3609 — APPROVED** [review]
Thanks Leigh!!

**Pull request merged**